script/007_DeployColdStorageAddressBookPlugin.s.sol

```diff
@@ -18,8 +18,9 @@
  */
 pragma solidity 0.8.24;
 
-import {ColdStorageAddressBookPlugin} from
-    "../src/msca/6900/v0.7/plugins/v1_0_0/addressbook/ColdStorageAddressBookPlugin.sol";
+import {
```

Comment:
This actually update the source code, could we please look into updating https://github.com/circlefin/buidl-wallet-contracts/blob/entry-point-v0.6/foundry.toml to avoid this change in deployment PR? I'm not sure if all the changes would result in contract address change, but I assume you've already done the deployment on the audited version (w/o src code changes).